sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharp/CSharpExplodedGraph.cs

```diff
@@ -551,6 +560,12 @@ protected override void VisitInstruction(ExplodedGraphNode node)
             EnqueueNewNode(newProgramPoint, newProgramState);
         }
 
+        private static void VisitRecursivePattern(RecursivePatternSyntaxWrapper recursivePattern)
```

Comment:
Although not really important at the moment anyway, since it does nothing, but the method should theoretically take a program state and return a program state, otherwise even if some constraints are set inside this method, it will never be reflected in the following SE paths.
But as it does nothing at the moment anyway, we can leave it as is if you want

Reply:
I think it makes sense to have the correct format. I've update the code.